README.md

```diff
@@ -45,9 +45,18 @@ A diagram of this running 5 steps on a single sample image may help explain how
 
 ## Installation
 
-This library works in the browser, in Node, and on the commandline (CLI), which we think is great. You can start a local environement to test the UI with `npm start`
+This library works in the browser, in Node, and on the commandline (CLI), which we think is great. You can setup a local environement to test the UI with `npm setup` followed by `npm start`
 On Unix based environments the command should be run as root user using `sudo npm start` and on windows the command should be run as administrator.
 
+Our npm scripts do not support windows shells, please run the following snippet in powershell.
+```powershell
```

Comment:
Could we have a subheader like `### Windows notes` here? Thanks!!!

Reply:
Nice idea!

Reply:
@aashna27 what do you think, could we perhaps add this header in here? Thank you!